extensions/capability-negotiation.md

```diff
@@ -343,6 +383,10 @@ change the capabilities of the client until the last ACK of the set has been sen
 
 ### The CAP NAK subcommand
 
+The format of a `CAP NAK` message is:
+
+    CAP <nick> NAK :[[-]<extension 1> ... [[-]<extension n>]]
```

Comment:
I don't think the spec allows dashes here: compare "The last parameter is a space-separated list of capabilities." with the wording from CAP ACK where dashes are explicitly allowed.

Reply:
I guess it would be pretty silly to allow a server to explicitly refuse disabling a capability.

Reply:
Unless it wasn't already requested, or is not a requestable cap (e.g. sts)

Reply:
Possibly? The protocol does explicitly allow for erroneous REQ entries, but I think the server would always be required to ACK them.

> If a client requests a capability which is already enabled, or tries to disable a capability which is not enabled, the server MUST continue processing the REQ subcommand as though handling this capability was successful.